<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgQTableWidget</class>
 <widget class="QDialog" name="dlgQTableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demo del Control QTableWidget</string>
  </property>
  <widget class="QLabel" name="lblRegistro">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Total de Registros</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTotal">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblData">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>961</width>
     <height>721</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
